src/theme.c

```diff
@@ -164,7 +167,13 @@ create_hover_fallback(struct theme *theme, const char *icon_name,
 	float overlay_color[4] = { 0.15f, 0.15f, 0.15f, 0.3f};
 	enum corner corner = corner_from_icon_name(icon_name);
 
-	if (corner == LAB_CORNER_UNKNOWN) {
+	if (theme->window_button_hover_bg_shape == LAB_CIRCLE) {
```

Comment:
I could turn this into a switch statement to make it more obvious that we are handling two different shapes here.

Reply:
Yes. I think that is a good idea.